compiler/luci-interpreter/src/kernels/Equal.cpp

```diff
@@ -49,6 +49,19 @@ void Equal::execute() const
     case DataType::FLOAT32:
       evalFloat();
       break;
+    case DataType::S64:
+      evalInteger<int64_t>();
+      break;
+    case DataType::S32:
+      evalInteger<int32_t>();
+      break;
+    case DataType::U64:
+      evalInteger<uint64_t>();
+      break;
+    case DataType::U32:
+      evalInteger<int32_t>();
```

Comment:
maybe `uint32_t`

Reply:
I skipped this case when made #8157
I'll add it as a separate change